doc/man3/CMS_add1_signing_cert_v2.pod

```diff
@@ -0,0 +1,45 @@
+=pod
+
+=head1 NAME
+
+CMS_add1_signing_cert_v2 - add ESS signing-certificate V2 signed
+attribute to a CMS_SignerInfo data structure
```

Comment:
I see no point having `CMS_add1_signing_cert` and `CMS_add1_signing_cert_v2` described in separate files.  Just make sure both names are mentioned in `NAME`, and our installation process will make sure to create the necessary symlinks.

Reply:
I did not understand, in which other files should these functions be described?

Reply:
He meant: use a single file CMS_add1_signing_cert.pod for both. Otherwise you need to duplicate a lot of text.